frontends/web/src/routes/settings/settings.tsx

```diff
@@ -169,7 +166,7 @@ class Settings extends Component<Props, State> {
                     <Header title={<h2>{t('settings.title')}</h2>}>
                         {
                             !deviceIDs.length && (
-                                <Link onClick={this.backHome} className="flex flex-row flex-items-center">
+                                <Link to='/' className="flex flex-row flex-items-center">
```

Comment:
Bug: if no devices are connected, there should be a home button in the header on the settings page but this isn't rendered. Gutfeeling: this is the same issue as why account-overview doesn't show account-info link.

@BrianCraig could you take a look?

Reply:
#1539